service-api/module/Application/src/Controller/StatusController.php

```diff
@@ -248,7 +226,7 @@ public function get($ids)
                         $rejectDate = isset($lpaDetail['rejectedDate']) ? $lpaDetail['rejectedDate'] : null;
 
                         // If it doesn't match what we already have update the database
-                        if (!is_null($lpaDetail['status']) && $lpaDetail['status']){
+                        if (!is_null($lpaDetail['status']) && $lpaDetail['status'] && $lpaDetail['status'] != $currentProcessingStatus){
```

Comment:
it's possible to change this line to read 
`if (isset($lpaDetail['status']) && $lpaDetail['status'] !== $currentProcessingStatus) {`

(untested)

Note: always do an explicit test `!==`